### PR TITLE
fix perf_speedy report

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
The report currently chokes on quotes in the commit message (see 550aa48fc52cb6dfa19de6b86e27b481d3835048). Rather than trying to correctly excape things in Bash, this PR delegates the quote handling to jq, because having to deal with Bash embedded in YAML is hard enough.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: ci/cron/daily-compat.yml
Comment (cocreature) [inline ci/cron/daily-compat.yml:26]: I’m confused, why is there so much deleted stuff?

Comment (garyverhaegen-da) [reply ci/cron/daily-compat.yml:26]: Second commit will be dropped before merging; only the first one needs review. Sorry, should have mentioned that.

Comment (garyverhaegen-da) [reply ci/cron/daily-compat.yml:26]: Well actually, that failed, so hold off on reviewing.

**Comment by garyverhaegen-da** [comment]
Oh, forgot to update: this is now fully working and thus ready for review. You can see there's a quote in the commit title and the compat tests (and thus the perf) have run successfully.

**Review by cocreature — APPROVED** [review]
Thanks!

**Pull request merged**
